### DISCUSSION TITLE
Problem in splitting the domain

### DISCUSSION THREAD

**Discussion by khushbuiitk** [idaholab/moose#27573] [Q&A Meshing]
Hi MOOSE community, 

I am trying to split the domain into two parts and assign different properties like conductivity to each split domain to check if it grows faster. I tried using a subdomain generator, but it isn't working at all. 

Please help me with this.
Thanks

**Comment by GiudGiud** [comment]
Hello

Can you describe the split of the domain with simple expressions of the local coordinates? For example x > 2 for the other subdomain.
If so, you could use this mesh generator to do the split?
https://mooseframework.inl.gov/source/meshgenerators/ParsedSubdomainMeshGenerator.html

Maybe paste the Mesh block here if you cannot get it to work

Guillaume
